src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -43,35 +47,67 @@
 
     @RequestMapping("/")
     public String greeting(Model model) {
-        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());  // inject in DOM for AJAX etc
+        // inject
+        // in
+        // DOM
+        // for
+        // AJAX
+        // etc
```

Comment:
why everywhere comments changed to one word per line?

Reply:
I think the eclipse style formatter that we use does this. Should i "overrule" the formatter or leave this?

Reply:
Better do as it was before. As for me, I'm not sure if these comments are still needed there

Reply:
I guess i´ll disable the formatter then as it does this automatically, i completely agree with you.